Helper/TokenHelper.php

```diff
@@ -344,6 +346,14 @@ private function addHelper($type)
 
                                 return $dash ? substr((string) $value, 0, $dash) : $value;
                             },
+                            'plus4' => function ($value) {
+                                preg_match('/\d-([\d]{4})/', $value, $matches);
```

Comment:
if this is for the delivery route digits, should it except both "-" and "+" ?

Reply:
\d+[-+]([\d]{4})

Reply:
I would say yes... but we don't have delivery route digits in our systems.